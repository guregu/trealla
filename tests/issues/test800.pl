:- initialization(main).

main :-
	number_chars(N,"%\n(7").
