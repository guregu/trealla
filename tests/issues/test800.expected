   Warning: Initialization goal exception: error(syntax_error(unexpected_char),number_chars/2).
